Userland/Utilities/mount.cpp

```diff
@@ -197,19 +199,60 @@ static ErrorOr<void> print_mounts()
     return {};
 }
 
+static ErrorOr<void> apply_file_system_specific_options(int mount_fd, Vector<String> const& filesystem_specific_options)
+{
+    for (auto& option : filesystem_specific_options) {
+        auto key_value_option = TRY(option.split_limit('=', 2));
+        MountSpecificFlag flag;
+        flag.key_string_length = key_value_option[0].bytes().size();
+        flag.key_string_addr = key_value_option[0].bytes().data();
+        if (key_value_option.size() > 1 && !key_value_option[1].is_empty()) {
+            auto possible_int_value = AK::StringUtils::convert_to_int<i64>(key_value_option[1]);
+            auto possible_uint_value = AK::StringUtils::convert_to_uint<u64>(key_value_option[1]);
```

Comment:
```suggestion
            auto possible_int_value = key_value_option[1].to_number<i64>();
            auto possible_uint_value = key_value_option[1].to_number<u64>();
```
(And then remove the StringUtils include.)